packages/components/src/components/Image/Image.tsx

```diff
@@ -9,11 +8,7 @@ export interface ImageProps extends React.ImgHTMLAttributes<HTMLImageElement> {
 
 export const Image: React.FC<ImageProps> = ({ className = '', src, cfImageAsset, ...props }) => {
   if (!cfImageAsset && !src) {
-    return (
-      <div className="cf-no-image-wrapper">
-        <img src={placeholderImage} className={'cf-image ' + className} {...props} />
-      </div>
-    );
+    return <img src={placeholderImage} className={'cf-image cf-no-image' + className} {...props} />;
```

Comment:
Should there be an extra space after the `cf-no-image` class?

Reply:
good catch